package.json

```diff
@@ -54,13 +54,18 @@
     }
   },
   "dependencies": {
+    "@rollup/plugin-babel": "^6.0.4",
```

Comment:
the previous versions of these plugins were named `rollup-plug-xxxx`. Rollup has since upgraded and changed the name of these plugins to the `@rollup/plugin-xxx` naming convention.